### PR TITLE
Install packages. Redux skeleton

### PR DESCRIPTION
**Pull request by pashura**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -7,12 +7,19 @@
     "@testing-library/jest-dom": "^5.11.6",
     "@testing-library/react": "^11.2.2",
     "@testing-library/user-event": "^12.2.2",
+    "bootstrap": "^4.5.3",
+    "jquery": "^3.5.1",
```
Comment (kkrisstoff) [inline package.json:11]: what??

File: package.json [hunk 2]
```diff
@@ -7,12 +7,19 @@
...
     "react": "^17.0.1",
+    "react-bootstrap": "^1.4.0",
     "react-dom": "^17.0.1",
+    "react-redux": "^7.2.2",
+    "react-router-dom": "^5.2.0",
     "react-scripts": "4.0.1",
     "redux": "^4.0.5",
     "redux-saga": "^1.1.3",
-    "web-vitals": "^0.2.4"
+    "redux-thunk": "^2.3.0"
```
Comment (kkrisstoff) [inline package.json:22]: you might not need it

File: src/containers/App.js [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+import TestDataArea from "../components/TestDataArea";
+import ErrorsArea from "../components/ErrorsArea";
+
+
+class App extends Component {
+
+    render() {
+        return (
+            <div className="container">
+                <div className="row">
```
Comment (kkrisstoff) [inline src/containers/App.js:15]: looks like you should use bootstrap-react components

File: src/containers/Root.js [hunk 4]
```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { Provider } from 'react-redux'
+import { Route } from 'react-router-dom'
+import App from './App'
+
+const Root = ({ store }) => (
+    <Provider store={store}>
+        <div>
+            <Route path="/" component={App} />
```
Comment (kkrisstoff) [inline src/containers/Root.js:10]: i'd like to avoid common names here

File: package.json (hunk 1 above)
Comment (pashura) [reply package.json:11]: `bootstrap` requires `jquery` to be installed 🤷‍♂️

File: src/containers/App.js (hunk 3 above)
Comment (pashura) [reply src/containers/App.js:15]: I was thinking about choosing between `bootstrap-react` and `bootstrap`

File: src/containers/Root.js (hunk 4 above)
Comment (kkrisstoff) [reply src/containers/Root.js:10]: do you need jQuery for `bootstrap-react`? ))

Comment (pashura) [reply src/containers/Root.js:10]: nice catch

**Pull request merged**